docs/methods/search_parameter_optimization.md

```diff
@@ -0,0 +1,131 @@
+
+The process of choosing the best batch step for calibration and optimization is illustrated in the figure below, which shows optimization and calibration over seven iterations; the batch size is increased in the first iteration until sufficient precursors are detected, and subsequently reduced when the proportion is sufficiently high that a previous step should reach the target as well; if, however, fluctuations in the number of identifications mean that not enough precursors are actually identified, the next step in the batch plan will be searched as well to ensure that calibration is always performed on at least the target number of precursors.
+
+<img src="../_static/images/methods_optimization_calibration.png" width="100%" height="auto">
+
+### Calibration
+If enough confident target precursors have been detected, they are calibrated to the observed values using locally estimated scatterplot smoothing (LOESS) regression. For calibration of fragment m/z values, a fixed number of up to 5000 of the best fragments according to their  XIC correlation are used. For precursor calibration all precursors passing 1% FDR are used. Calibration is performed prior to every optimization.
+
+### Optimization
+For optimizing the search space, tolerances like retention time, ion mobility and m/z ratios need to be reduced. The goal is to cover the expected spectrum space but reduce it as much as possible to accelerate search and gain statsitical power. Search starts with initial tolerances as defined in `search_initial`.  For targeted optimization, the 95% deviation after calibration is adopted as the new tolerance until the target tolerances defined in the `search` section are reached. For automatic optimization, the 99% deviation plus 10% of the absolute value of the tolerance is adopted as the new tolerance, and search continues until parameter-specific convergence rules are met.
+
+The optimization is finished as soon as the minimum number of steps `min_steps` has passed and all tolerances have either 1) reached the target tolerances defined in `search` if using targeted optimization, or 2) have converged if using automatic optimization.
+
+## Configuring calibration and optimization
+The configuration below will perform targeted optimization of precursor m/z, fragment m/z and ion mobility, and automatic optimization of retention time.
+
+```yaml
```

Comment:
Showing the config here is a good service to the reader, but it is likely that this will get outdated .. 
you may choose to add all these comments to the actual default config, and just point to the `calibration`, `search_initial`, and `search` keys here..
but it's also fine to leave it like this
(I had this discussion with @GeorgWa already ;-))

Reply:
I've simplified what I'm showing so as to include only relevant sections of each field in the config. I'm not sure that this fully addresses the issue you raise, but I think it makes it slightly better than before.